server/src/lib/server.rs

```diff
@@ -361,6 +362,32 @@ async fn instance_ensure(
     Ok(HttpResponseCreated(api::InstanceEnsureResponse {}))
 }
 
+#[endpoint {
+    method = GET,
+    path = "/instances/{instance_id}/uuid",
+    unpublished = true,
```

Comment:
cc @smklein had to add this endpoint to make the tool a bit more ergonomic to allow for interacting with instances by name instead of uuid.

Reply:
Seems reasonable; this is trivial to maintain in the server so that tradeoff seems worth it.